operations/fetch.go

```diff
@@ -263,16 +264,28 @@ func cloneSource(task *service.RestTask, project *model.ProjectRef, config *mode
 	if variant == nil {
 		return errors.Errorf("couldn't find build variant '%v' in config", task.BuildVariant)
 	}
+	mfest, err := manifest.FindFromVersion(task.Version, task.Project, task.Revision)
```

Comment:
Since this runs on a machine that's not the application server, we can't make db queries. You'll need to query the API to get this data